packages/angular_devkit/build_angular/src/webpack/plugins/any-component-style-budget-checker.ts

```diff
@@ -42,30 +44,50 @@ export class AnyComponentStyleBudgetChecker {
           '.sass',
         ];
 
-        const componentStyles = Object.keys(compilation.assets)
-            .filter((name) => cssExtensions.includes(path.extname(name)))
-            .map((name) => ({
-              size: compilation.assets[name].size(),
-              label: name,
-            }));
+        const componentStyles = Object.keys(_compilation.assets)
+          .filter((name) => cssExtensions.includes(path.extname(name)))
+          .map((name) => ({
+            size: _compilation.assets[name].size(),
+            label: name,
+          }));
         const thresholds = flatMap(this.budgets, (budget) => calculateThresholds(budget));
 
-        for (const { size, label } of componentStyles) {
-          for (const { severity, message } of checkThresholds(thresholds[Symbol.iterator](), size, label)) {
+        for (const {size, label} of componentStyles) {
+          for (const {severity, message} of checkThresholds(thresholds[Symbol.iterator](), size, label)) {
             switch (severity) {
               case ThresholdSeverity.Warning:
-                addWarning(compilation, message);
+                addWarning(_compilation, message);
                 break;
               case ThresholdSeverity.Error:
-                addError(compilation, message);
+                addError(_compilation, message);
                 break;
               default:
                 assertNever(severity);
                 break;
             }
           }
         }
-      });
+      };
+
+      if (isWebpackFiveOrHigher()) {
+        // question to @alan-agius4 :
+        // there are will be 5 PRs with very similar changes, should I add comments regarding source of info like below:
+        // webpack 5 migration "guide"
+        // https://github.com/webpack/webpack/blob/07fc554bef5930f8577f91c91a8b81791fc29746/lib/Compilation.js#L535-L539
```

Comment:
A comment never hurt anyone! So I am good with adding a link to the source code.

Reply:
@alan-agius4  awesome, thanks